src/Constants/hiddenCategories.js

```diff
@@ -0,0 +1,3 @@
+export const hiddenCategories = {
+    "hiddenCategories": [25]
```

Comment:
Instead of using hidden categories twice, just do this

export const hiddenCategories = [25]

Reply:
Done